pages/assessment.tsx

```diff
@@ -93,7 +93,7 @@ const Assessment: NextPage = () => {
     const maxReAssessmentPercentage = 0.2;
     if (
       // not currently assessing a reAssessment
-      currentPage * (maxItemsPerPage - 1) <
+      currentPage *
```

Comment:
this is always true

Reply:
The power of code review! Nice catch! Was just a weird mistake I think